marketing_emails/email_kickoff_keynote.md

```diff
@@ -0,0 +1,39 @@
+*Fellow Pythonistas!*
+
+It is my honor to announce that PyTN 2018 is on its way.  The 2018 event will take place on February 10th and 11th at the Nashville Law School.  The organizers are working on setting up the 2018 event to meet the level of excellence former chairman Jason Myers set.
+
+We can first announce the keynote speakers who are sure to give you reason enough to put PyTN 2018 on your calendar:
```

Comment:
Maybe something more like: "And if that wasn't enough reason to come, we have just announced our keynoters for PyTN 2018!: ..."